PT/asciidoc/src/05_building_block_view.adoc

```diff
@@ -0,0 +1,225 @@
+
+
+== Visão de Blocos de Construção
+
+ifdef::arc42help[]
+[role="arc42help"]
+****
+.Conteúdo
+A visão de blocos de construção mostra a decomposição estática do sistema em blocos de construção (módulos, componentes, subsistemas, classes, interfaces, pacotes, bibliotecas, frameworks, camadas, partições, níveis, funções, macros, operações, estruturas de dados, ...) bem como suas dependências (relacionamentos, associações, ...)
+
+Esta visão é obrigatória para toda documentação de arquitetura.
+Em analogia a uma casa, esta é a _planta baixa_.
+
+.Motivação
+Mantenha uma visão geral do seu código-fonte tornando sua estrutura compreensível por meio de
+abstração.
+
+Isso permite que você se comunique com suas partes interessadas em um nível abstrato sem revelar detalhes de implementação.
+
+.Forma
+A visão de blocos de construção é uma coleção hierárquica de caixas pretas e caixas brancas
+(veja a figura abaixo) e suas descrições.
+
+image::05_building_blocks-EN.png["Hierarquia de blocos de construção"]
+
+*Nível 1* é a descrição da caixa branca do sistema geral, juntamente com descrições
+de caixa preta de todos os blocos de construção contidos.
+
+*Nível 2* amplia alguns blocos de construção do nível 1.
+Portanto, ele contém a descrição da caixa branca dos blocos de construção selecionados do nível 1, juntamente com descrições de caixa preta de seus blocos de construção internos.
+
+*Nível 3* amplia os blocos de construção selecionados do nível 2, e assim por diante.
+
+
+.Mais informações
+
+Consulte https://docs.arc42.org/section-5/[Building Block View] na documentação do arc42.
+
+****
+endif::arc42help[]
+
+=== Visão Sistêmica Geral de Caixa Branca
+
+ifdef::arc42help[]
+[role="arc42help"]
+****
+Aqui você descreve a decomposição geral do sistema usando o seguinte modelo de caixa branca. Ele contém
+
+* um diagrama de visão geral
+* uma motivação para a decomposição
+* descrições de caixa preta dos blocos de construção contidos. Para isso, oferecemos alternativas:
+
+** use _uma_ tabela para uma visão geral curta e pragmática de todos os blocos de construção contidos e suas interfaces
+** use uma lista de descrições de caixa preta dos blocos de construção de acordo com o modelo de caixa preta (veja abaixo).
+Dependendo da sua escolha de ferramenta, esta lista pode ser subcapítulos (em arquivos de texto), subpáginas (em um Wiki) ou elementos aninhados (em uma ferramenta de modelagem).
+
+
+* (opcional:) interfaces importantes, que não são explicadas nos modelos de caixa preta de um bloco de construção, mas são muito importantes para entender a caixa branca.
+Já que há tantas maneiras de especificar interfaces, por que não fornecer um modelo específico para elas?
+No pior caso, você tem que especificar e descrever sintaxe, semântica, protocolos, tratamento de erros,
+restrições, versões, qualidades, compatibilidades necessárias e muito mais.
+Na melhor das hipóteses, você conseguirá usar exemplos ou assinaturas simples.
```

Comment:
Maybe remove the "ou assinaturas simples"

Reply:
Changed to "ou descrições simples"